### DISCUSSION TITLE
How can I implement skip connections between GNN layer?

### DISCUSSION THREAD

**Discussion by ecastillot** [stellargraph/stellargraph#1985] [Q&A]
I want to implement skip connections between GNN layer. For example,  how can I make the next structure?. 
![skip](https://user-images.githubusercontent.com/63883182/136724274-d29eefe4-ae6b-44c4-9e5d-6826c5d6706a.PNG)

Can anyone help me with any ideas?
Thanks

**Comment by mcembalest** [comment]
Same question here!
